src/table/src/EditableCell.js

```diff
@@ -96,30 +87,32 @@ class EditableCell extends React.PureComponent {
      * When the user presses a character on the keyboard, use that character
      * as the value in the text field.
      */
-    if (key.match(/^[a-z]{0,10}$/) && !e.metaKey && !e.ctrlKey && !e.altKey) {
-      this.setState({
-        isEditing: true,
-        value: key
-      })
-    } else if (key === 'Enter') {
+    if (key === 'Enter' || key === 'Shift' || this.state.value !== '') {
```

Comment:
Why add this `key === 'Shift' || this.state.value !== ''`?

Reply:
`this.state.value`: The functionality that I was trying to preserve was on keydown, if there was already text in the cell, instead of updating the cell immediately, to enter editing mode. 
`Shift`: Currently, the cells only take in a-z. This lets the user also input A-Z (changing the regex to [a-zA-z] automatically enters the word 'Shift' in the field). I thought this would be a nice feature, but I can take it off.

Reply:
Do we have a storybook or example in the docs for this?